staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_type.go

```diff
@@ -492,7 +492,7 @@ func (c *$.type|privatePlural$) List(opts $.ListOptions|raw$) (result *$.resultT
 			return
 		}
 
-		if err != nil && $.stringsContains|raw$(err.Error(), "forbidden") &&
+		if err != nil && $.errorForbidden|raw$(err) &&
 			$.stringsContains|raw$(err.Error(), "no relationship found between node") {
 			$.klogInfof|raw$(6).Infof("Skip error %v in list", err)
```

Comment:
Add more info about the resources to list in the log message.

Reply:
This is generic list 

> I like this change.
> 
> Would like to see some similar changes to have a clean way to check whether the error contains the string ""no relationship found between node".

The error message is in authorization plug in. You might want to ask Peng Du to make the changes.